flake.nix

```diff
@@ -1,7 +1,9 @@
 {
   description = "ROS messages which correspond to the loco-3d/linear-feedback-controller package.";
 
-  inputs.nix-ros-overlay.url = "github:lopsided98/nix-ros-overlay/master";
+  # [2025/03/25] Target develop as main branch is too unstable for now.
+  # Keep it until next release on jazzy.
```

Comment:
Well, I guess we can default to develop on all our repos. This is already in gepetto/nix, and we clearly don’t fall in master use cases, and it is usually too old for us, when it is not downright broken.
```suggestion
```